integration-tests/environments/node/index.mjs

```diff
@@ -21,13 +21,19 @@ import { Client } from "mocha-remote-client";
 import fs from "fs-extra";
 import path from "path";
 
+import v8 from "node:v8";
+import vm from "node:vm";
+
+v8.setFlagsFromString("--expose_gc");
```

Comment:
I got some inspiration for this from https://github.com/legraphista/expose-gc/blob/master/function.js